tm2py/config.py

```diff
@@ -1152,6 +1151,35 @@ class TransitClassConfig(ConfigItem):
     required_mode_combo: Optional[Tuple[str, ...]] = Field(default=None)
 
 
+@dataclass(frozen=True)
+class ManualJourneyLevelsConfig(ConfigItem):
+     """Manual Journey Level Specification"""
+     level_id: int
+     group_fare_systems: Tuple[int, ...]
+
+
+@dataclass(frozen=True)
+class TransitJourneyLevelsConfig(ConfigItem):
+    """Transit manual journey levels structure."""
+    use_algorithm: bool = True
```

Comment:
@vivverma9 
Add a docstring specific to `use_algorithm` for documentation. Something like:

```{python}
"""
The original translation from Cube to Emme used an algorithm to, as faithfully as possible, reflect transfer fares via journey levels. The algorithm examines fare costs and proximity of transit services to create a set of journey levels that reflects transfer costs. While this algorithm works well, the Bay Area's complex fare system results in numerous journey levels specific to operators with low ridership. The resulting assignment compute therefore expends a lot of resources on these operators. Set this parameter to `True` to use the algorithm. Either `use_algorithm` or `specify_manually` must be `True`. 
"""